src/ucp/core/ucp_ep.c

```diff
@@ -3149,8 +3149,20 @@ static ucs_status_t ucp_ep_query_sockaddr(ucp_ep_h ep, ucp_ep_attr_t *attr)
     return UCS_OK;
 }
 
+static ucs_status_t ucp_ep_query_debug_str(ucp_ep_h ep, ucp_ep_attr_t *attr)
+{
+    if (attr->debug_str_max == 0ul) {
+        return UCS_ERR_INVALID_PARAM;
+    }
```

Comment:
no need to fail - just do nothing

Reply:
in this case string can be invalid, IMO this is invalid argument